win32ss/user/ntuser/tags.h

```diff
@@ -71,6 +71,7 @@
 #define GDITAG_DC_FREELIST               'fcdG'
 #define GDITAG_DWMSTATE                  'scDG'
 #define GDITAG_DEVMODE                   'vedG'
+#define GDITAG_MDEV                      'veDG'
```

Comment:
has same value as the PDEV tag; isn't it possible to get something like `'vdmG'`?

Reply:
Done